pkg/network/setup/netstat.go

```diff
@@ -90,60 +90,71 @@ func (c *NetStat) UpdateStatus(vmi *v1.VirtualMachineInstance, domain *api.Domai
 		return nil
 	}
 
-	multusStatusNetworksByName := netvmispec.IndexInterfaceStatusByName(
+	oldIfaceStatusesByName := netvmispec.IndexInterfaceStatusByName(
```

Comment:
nit: `old` is so negative... probably because I am old :)

Maybe prefix with `orig`or `prev`.

Reply:
This change was reverted.
Nevertheless, `prev` is used instead of `old` in applicable places.